// api/mockCategories.js
export const categories =

{
    phones: [
        {
            id: "1",
            name: 'Apple'
        },
        {
            id: "2",
            name: 'Samsung'
        },
        {
            id: "3",
            name: 'Htc'
        },
        {
            id: "4",
            name: 'Lenovo'
        },
        {
            id: "5",
            name: 'Microsoft'
        }
    ],
    laptop: [
        {
            id: "6",
            name: 'Apple'
        },
        {
            id: "7",
            name: 'Asus'
        },
        {
            id: "8",
            name: 'Dell'
        },
        {
            id: "9",
            name: 'HP'
        },
        {
            id: "10",
            name: 'Xiaomi'
        }
    ],
    tv: [
        {
            id: "11",
            name: 'LG'
        },
        {
            id: "12",
            name: 'SAMSUNG'
        },
        {
            id: "13",
            name: 'SONY'
        },
        {
            id: "14",
            name: 'PHILIPS'
        },
        {
            id: "15",
            name: 'PANASONIC'
        }
    ],
    monitor: [
        {
            id: "16",
            name: 'Dell'
        },
        {
            id: "17",
            name: 'Samsung'
        },
        {
            id: "18",
            name: 'LG'
        },
        {
            id: "19",
            name: 'HP'
        },
        {
            id: "20",
            name: 'Philips'
        }
    ],
    tablet: [
        {
            id: "21",
            name: 'ASUS'
        },
        {
            id: "22",
            name: 'HP'
        },
        {
            id: "23",
            name: 'Huawei'
        },
        {
            id: "24",
            name: 'Lenovo'
        },
        {
            id: "25",
            name: 'Samsung'
        }
    ]
}
